Modix.Bot/Extensions/GuildChannelExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Discord;
+
+namespace Modix.Bot.Extensions
+{
+    internal static class GuildChannelExtensions
+    {
+        public static bool IsPublic(this IGuildChannel channel)
+        {
+            if (channel?.Guild is IGuild guild)
+            {
+                var isDefaultPerms = guild.Features;
```

Comment:
It doesn't look like this is used anywhere

Reply:
Whoops! Good catch. I added it just so I could inspect it while debugging but somehow it ended up in my commit. I'll remove it.